src/main/java/net/openhft/chronicle/wire/BinaryWire.java

```diff
@@ -439,6 +444,14 @@ private void copyOne(@NotNull WireOut wire, boolean first) throws InvalidMarshal
         }
     }
 
+    private void copyHistoryMessage(@NotNull WireOut wire) {
+        VanillaMessageHistory vmh = new VanillaMessageHistory();
```

Comment:
Does this need to be pooled or pulled into a field reference and recycled?

Reply:
Agree